index.js

```diff
@@ -1,50 +1,55 @@
-const regex = require('./regex.js')
-
-module.exports = BananaSlug
+import { regex } from './regex.js'
 
 const own = Object.hasOwnProperty
 
-function BananaSlug () {
-  const self = this
-
-  if (!(self instanceof BananaSlug)) return new BananaSlug()
```

Comment:
This isn’t possible with actual classes anymore: they must be constructed with `new`

Reply:
This code smells like old books. :)